.gitignore

```diff
@@ -33,3 +33,8 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+# Ignore database config.
+/config/database.yml
+
+/Gemfile.lock
```

Comment:
Interesting, it's already part of the updated main branch, did you fetch the new changes? :)